pkg/occlient/occlient.go

```diff
@@ -470,6 +471,25 @@ func (c *Client) GetProjectNames() ([]string, error) {
 	return projectNames, nil
 }
 
+func (c *Client) GetProject(projectName string) (*projectv1.Project, error) {
+	prj, err := c.projectClient.Projects().Get(projectName, metav1.GetOptions{})
+	if err != nil {
+		istatus, ok := err.(kerrors.APIStatus)
+		if ok {
+			status := istatus.Status()
+			// the
```

Comment:
this comment can probably be deleted ;-)

Reply:
yeah, will do

Reply:
done